src/OrleansAWSUtils/Providers/Storage/DynamoDBStorage.cs

```diff
@@ -0,0 +1,334 @@
+﻿using Newtonsoft.Json;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.Serialization;
+using OrleansAWSUtils;
+using OrleansAWSUtils.Storage;
+using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Orleans.Storage
+{
+    /// <summary>
+    /// AWS DynamoDB basic Storage provider
+    /// </summary>
+    public class DynamoDBStorage : IStorageProvider
```

Comment:
Why storage provider is part of membership table pull request? I thought you had a separate pr for it.